resdk/resolwe.py

```diff
@@ -218,13 +221,13 @@ def _upload_tools(self, tools):
         if TOOLS_REMOTE_HOST is None:
             raise ValueError("Define TOOLS_REMOTE_HOST environmental variable")
 
-        print("SCP: {}".format(TOOLS_REMOTE_HOST))
+        self.logger.info("SCP: %s", TOOLS_REMOTE_HOST)
         for tool in tools:
             status = subprocess.call('scp -r {} {}'.format(tool, TOOLS_REMOTE_HOST), shell=True)
```

Comment:
If `self.logger`'s level is above info, you should suppress the standard output of this command. Look [here](https://github.com/genialis/resolwe/blob/master/resolwe/flow/tests/test_executors.py#L35) how to do that. Or you can find a way to "pipe" the stdout and stderr to the logger directly.